pkg/cmd/extension/command.go

```diff
@@ -347,7 +347,11 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 						return err
 					}
 
-					if err := m.Install(repo, pinFlag); err != nil {
+					io.StartProgressIndicator()
+					err = m.Install(repo, pinFlag)
+					io.StopProgressIndicator()
```

Comment:
Couple of thoughts:

1. Should we have a label for the indicator to help those who use screen readers know what is going on?
2. Generally recommend `defer`ing the cessation of the indicator
   ```
					   io.StartProgressIndicator()
					   defer io.StopProgressIndicator()
					   err = m.Install(repo, pinFlag)
   ```